ProcessLib/InitialCondition.h

```diff
@@ -70,7 +70,8 @@ class MeshPropertyInitialCondition : public InitialCondition
 		assert(_property.getMeshItemType() == MeshLib::MeshItemType::Node);
 	}
 
-	virtual double getValue(MeshLib::Node const& n) const override
+	virtual double getValue(MeshLib::Node const& n,
+	                        int const component_id) const override
```

Comment:
Yes. It's not used yet, because, well, it's not implemented. Will make another PR, because it needs a config tree's ability to read tuples...